securecn/resource_portshift_k8s_cluster.go

```diff
@@ -39,6 +39,7 @@ const CiImageValidationFieldName = "ci_image_validation"
 const RestrictRegistries = "restrict_registries"
```

Comment:
nice work!
please update the provider readme (in file k8s_cluster.md) and talk to david about updating the official doc here https://appsecurity.readme.io/docs/terraform